addons/web_editor/static/src/components/media_dialog/video_selector.js

```diff
@@ -256,7 +256,7 @@ export class VideoSelector extends Component {
             div.innerHTML = `
                 <div class="css_editable_mode_display"></div>
                 <div class="media_iframe_video_size" contenteditable="false"></div>
-                <iframe frameborder="0" contenteditable="false" allowfullscreen="allowfullscreen"></iframe>
+                <iframe loading="lazy" frameborder="0" contenteditable="false" allowfullscreen="allowfullscreen"></iframe>
```

Comment:
Question: here it sets as lazy the video uploaded when replacing an image for example, but it does not do it for the iframe of a background video. Shouldn't the background videos be lazy too ? :slightly_smiling_face:

Reply:
Yep, I didn't think about the BG videos. It's done.